src/machines/modelingMachine.ts

```diff
@@ -1689,6 +1658,46 @@ export const modelingMachine = setup({
         }
       }
     ),
+    deleteSelectionAstMod: fromPromise(
```

Comment:
Didn't change much here but took the chance to make it promise based like other newer mods